### DISCUSSION TITLE
Add External sealed secret CRD

### DISCUSSION THREAD

**Discussion by Deahan-We** [bitnami-labs/sealed-secrets#1249] [Ideas]
Hi.

First of all, I'm a big fan of Sealed Secrets, but I think it has a weakness in a multiple-cluster environment.

Our company has a few clusters that need to use Sealed Secrets, but it really annoys me when I have to add a new Sealed Secret for a new cluster.

So, I suggest an External Sealed Secret (whatever its name may be) CRD, similar to Argo CD's external cluster, for deploying applications to external clusters.

This External Sealed Secret would contain the external cluster configuration for deploying Kubernetes secrets and actual secret values.

Br
Jeff

**Comment by agarcia-oss** [comment]
Hi @Deahan-We thanks for the suggestion. I agree managing multiple clusters is not Sealed Secrets strong point. We're working at the moment on providing commercial support for that feature, I'll keep you posted with future developments!

Regards
